src/test-workspace.ts

```diff
@@ -105,26 +102,28 @@ export async function testWorkspace(
     localConversionMap: Map<string, string>, options: WorkspaceTestSettings) {
   const allRepos = options.reposToConvert;
 
-  logStep(1, 5, '🔧', `Preparing Repos...`);
+  logStep(1, 4, '🔧', `Preparing Repos...`);
```

Comment:
want to make that `4` a variable?

Reply:
Eh it's pretty minor, if we have to touch this again then we can do it